zerver/lib/markdown/__init__.py

```diff
@@ -1267,6 +1330,9 @@ def run(self, root: Element) -> None:
             else:
                 continue
 
+            if self.is_video(url):
+                self.handle_video_inlining(root, found_url)
```

Comment:
Is this missing a `continue`, so it doesn't fall through to the other cases below?

Reply:
Yeah.